tests/testthat/test-correlation.R

```diff
@@ -39,9 +39,9 @@ test_that('correlation works with two data tables', {
 test_that("correlation works with a single data table", {
   nSamples = 200
   testData1 <- data.table(
-    "contA1" = rnorm(nSamples),
-    "contB1" = rnorm(nSamples),
-    "contC1" = rnorm(nSamples)
+    "contA1" = rnorm(nSamples,1,.5), #sparcc wont like negative values
```

Comment:
We'd cover more cases if we left this as is and then sent an absolute value version of the data to tests with sparcc. Don't want to accidentally make a change later that breaks using negative values in the non-sparcc case!